basilisk/provider_engine/mistralai_engine.py

```
@@ -89,6 +142,17 @@ def models(self) -> list[ProviderAIModel]:
 				max_temperature=1.0,
 				default_temperature=0.7,
 			),
+			ProviderAIModel(
+				id="devstral-small-latest",
+				name="Devstral Small",
+				# Translators: This is a model description
+				description=_(
+					"A 24B text model, open source model that excels at using tools to explore codebases, editing multiple files and power software engineering agents."
```

Pylint (3.3.7)

[convention] Line too long (153/100)

(C0301)